.travis.yml

```diff
@@ -38,21 +38,27 @@ jobs:
       script: npm run test
 
     - stage: WDA build
-      # - name: iPhone SE, Xcode 9.2
-      # osx_image: xcode9.2
-      # env: IPHONE_MODEL="iPhone SE" IPAD_MODEL="iPad Air 2" IOS_VERSION="11.2" ACTION=build TARGET=runner
+    - name: iPhone 11, Xcode 11
```

Comment:
I think the very first item after `-stage` declaration should not have dash at the start, because it creates an extra job

Reply:
ah, maybe right https://docs.travis-ci.com/user/build-stages/